integrations/tensorboard/Neptune-TensorBoard.ipynb

```diff
@@ -0,0 +1,765 @@
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "%env NEPTUNE_API_TOKEN=ANONYMOUS\n",
+    "%env NEPTUNE_PROJECT=shared/tensorboard-integration"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "QoRgIu4tpcqW",
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "**Note:**   \n",
+    "Instead of logging data to the public project 'shared/tensorboard-integration' as an anonymous user 'neptuner' you can log it to your own project.\n",
+    "\n",
+    "To do that:\n",
+    "\n",
+    "1. Get your Neptune API token\n",
+    "\n",
+    "![image](https://neptune.ai/wp-content/uploads/get_token.gif)\n",
+    "\n",
+    "2. Pass the token to `NEPTUNE_API_TOKEN` environment variable\n",
+    "3. Create a new Neptune Project (learn how to do that [here](https://docs.neptune.ai/workspace-project-and-user-management/projects/create-project.html))\n",
+    "4. Pass your username and project_name to the `NEPTUNE_PROJECT` environment variable\n",
+    "\n",
+    "For example:\n",
+    "\n",
+    "```\n",
+    "%env NEPTUNE_API_TOKEN=eyJhcGlfYW908fsdf23f940jiri0bn3085gh03riv03irn\n",
+    "%env NEPTUNE_PROJECT=jakub-czakon/tensorboard_logs\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Z8of9J8662au",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 2: Convert TensorBoard logs to Neptune experiments"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "w9CfGImkP7JO",
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "Run the following command"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "CHBji8hq3Ngi",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! neptune tensorboard logs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "**Note:**  \n",
+    "You can specify the path to the directory where the 'mlruns' directory is. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## See converted experiments\n",
+    "\n",
+    "Click on the link(s) above to browse the TensorBoard run in Neptune or go to [shared/tensorflow-integration project](https://ui.neptune.ai/o/shared/org/tensorboard-integration/experiments?viewId=def2c858-3510-4bf9-9e52-8720fadecb11)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "g-Ae7n8l0R_j",
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "**Note:**  \n",
+    "Running the above code pushes all your TensorBoard logs to Neptune. You can rerun the code after running fresh experiments, and only they will be pushed."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "# Log new runs to Neptune via TensorBoard"
```

Comment:
Maybe it's good to mention that it work in real time, while previous method is for logging exps that are finished.

With CLI I can log new runs as well. I'm thinking if the title is misleading?